daemon/cmd/daemon_main.go

```diff
@@ -1338,6 +1341,11 @@ func initEnv(cmd *cobra.Command) {
 		option.Config.NeedsRelaxVerifier = true
 	}
 
+	if option.Config.LocalRouterIP != "" && option.Config.IPAM != "" {
+		option.Config.LocalRouterIP = ""
+		log.Warnf("Reset %s to empty because Cilium is handling IPAM.", option.LocalRouterIP)
```

Comment:
Let's use log.Fatal here instead. Users might miss the warning.

Reply:
Done.